index.ts

```diff
@@ -12,22 +12,14 @@ import type {
   FastifyPluginAsync,
   FastifyRequest,
   FastifyReply,
+  HTTPMethods,
 } from 'fastify';
 import type { RouteGenericInterface } from 'fastify/types/route';
 
-enum HTTPMethod {
-  GET = 'get',
-  POST = 'post',
-  PUT = 'put',
-  DELETE = 'delete',
-  OPTIONS = 'options',
-  PATCH = 'patch',
-  HEAD = 'head',
-  CONNECT = 'connect',
-  TRACE = 'trace',
-}
+const methods = ['DELETE', 'GET', 'HEAD', 'PATCH', 'POST', 'PUT', 'OPTIONS'];
```

Comment:
I see that HTTPMethods is a type and not an enum (unfortunately)
Can you add the type for this array? e.g. `const methods: HTTPMethods[]`?